### ISSUE TITLE
Remove testmondata automatically if old

### ISSUE THREAD

**Issue by jacksongoode** [tarpas/pytest-testmon#188]
Across versions we often get the error:
```
testmon.db.TestmonDbException: The stored data file ***/.testmondata version (0) is not compatible with current version (9). You must delete the stored data to continue.
```
To me, I would be happy with the default behavior of printing a warning regarding the old `testmondata` file, rename it to `testmondata.old` and generate anew. Currently we have to account for pytest failing as a result of this error when updating testmon. If this is something, you'd be open to I can make a PR.

**Comment by tarpas** [comment]
Incidentally I already worked on this as a part of a little refactoring. No PR needed, it should be released soon.

**Comment by tarpas** [comment]
This has been implemented since v1.3.4. 
@jacksongoode pytest-xdist support has been implemented in v1.4.0a0 (pre-release)
